File-Formats/DocIO/Convert-Word-Document-to-PDF-in-WinUI.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Convert Word document to PDF in WinUI | Syncfusion
+description: Convert Word document to PDF without Microsoft Word or interop dependencies in WinUI application using WinUI Word (DocIO) library
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word document to PDF in WinUI
+
+Syncfusion Essential DocIO is a [WinUI Word library](https://www.syncfusion.com/document-processing/word-framework/winui/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **create a Word document in WinUI**.
```

Comment:
Check and address feedbacks on "Essential DocIO", **create a Word document in WinUI**.